api/v2/guides/custom-publisher-integration.md

```diff
@@ -1,20 +1,22 @@
-[UID2 Overview](../../../README.md) > [Getting Started](../../README.md) > [v2](../README.md) > [Integration Guides](README.md) > Custom Publisher Integration Guide 
+[UID2 Overview](../../../README.md) > [Getting Started](../../README.md) > [v2](../README.md) > [Integration Guides](README.md) > Publisher Integration Guide, Server-Only (Without SDK)
 
-# Server-Only UID2 Integration Guide
+# Publisher Integration Guide, Server-Only (Without SDK)
 
 This guide is intended for app developers and CTV broadcasters who would like to generate identity tokens utilizing UID2 for the RTB bid stream, while integrating directly with UID2 rather than UID2-enabled single-sign-on or identity providers. 
 
-The guide outlines the [basic steps](#integration-steps) that you need to consider for your custom integration. For example, you need to decide how to implement user login and logout, how to manage UID2 identity information and use it for targeted advertising, how to refresh tokens, deal with missing identities, and handle user opt-outs. Here's an [example application](https://example-srvonly-integ.uidapi.com/) that demonstrates the workflow. For the application documentation, see [Server-Only UID2 Integration Example](https://github.com/UnifiedID2/uid2-examples/blob/main/publisher/server_only/README.md). See also [FAQs](#faqs).
+The guide outlines the [basic steps](#integration-steps) that you need to consider if you are building an integration without using an SDK. For example, you need to decide how to implement user login and logout, how to manage UID2 identity information and use it for targeted advertising, and how to refresh tokens, deal with missing identities, and handle user opt-outs. See also [FAQs](#faqs).
```

Comment:
I think we should mention somewhere that there are the following ways for a publisher to integrate:
1) client JS SDK with custom server code
2) client JS SDK with UID2 Java SDK on server
3) server-only integration: use UID2 Java SDK on server
4) server-only integration: use custom server code

This use-case is option (4), but readers wouldn't be aware they can use (3) as a better alternative

Reply:
Added this. It needs more detail later but it's there (tweaked the wording). Thx!